content/en/docs/setup/install/virtual-machine/index.md

```diff
@@ -194,64 +194,64 @@ Install Istio and expose the control plane so that your virtual machine can acce
 
 1. Create a template `WorkloadGroup` for the VM(s)
 
-    {{< tabset category-name="registration-mode" >}}
+{{< tabset category-name="registration-mode" >}}
```

Comment:
If you don't indent this 4 spaces, the numbered list won't work (the next list item will reset back to `1.`, instead of increment to `2.`).